_utilities/layout-grid.md

```diff
@@ -30,6 +28,63 @@ subnav:
   href: '#sass-mixins'
 - text: Latest updates
   href: '#changelog'
+spacing_settings:
```

Comment:
I do like the idea of only maintaining this information in one spot. Originally, I tried pulling in `spacing.yml` but since it pulled in border radius settings, I opted to create a table of the relevant settings instead.

I also considered using the liquid [slice filter](https://shopify.github.io/liquid/filters/slice/) to use `spacing.yml` but without the border-radius settings. 

```md
{% assign spacing_settings = site.data.settings.spacing.contents | slice: 3, 9 %}

{% include settings-table.html
  settings=spacing_settings
%}
```

Which outputs:

![image](https://github.com/uswds/uswds-site/assets/25211650/3d192823-7afe-4a9e-81bf-eae40db0fb86)

But this seems pretty fragile as well if `spacing.yml` changes at all.

My initial goal was to replace this code block:

![image](https://github.com/uswds/uswds-site/assets/25211650/f4fb5155-f14f-4928-8668-7229bca10947)

Do you think it would be valid to change the `#spacing-settings` jump link to link to the [spacing settings](https://designsystem.digital.gov/documentation/settings/#spacing-settings) on the settings page and not have this information on this page?

Reply:
I went and ahead and removed the `spacing_settings` front matter and implemented the slice method to showcase the idea but I'm not sure we'd want to rely on this solution.

https://github.com/uswds/uswds-site/blob/2872121df1a5ac2ccfafb062159c9f9f6e23329b/_utilities/layout-grid.md?plain=1#L363-L369

Take a look and let me know if we're better just removing this table and forwarding users to the settings page to find this information

Reply:
Removed in [245919e](https://github.com/uswds/uswds-site/pull/2276/commits/245919e23d7546be5ac0d183d516bab90252c694)